firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -60,6 +60,32 @@ class DictionaryEntryDetailSerializer(serializers.HyperlinkedModelSerializer):
     )
     categories = CategorySerializer(many=True)
     site = SiteSummarySerializer()
+    base_characters = serializers.SerializerMethodField()
+    words_in_phrase = serializers.SerializerMethodField()
+
+    @staticmethod
+    def get_base_characters(entry):
+        if "⚑" in entry.custom_order:
+            return "Due to this entry containing unknown characters, it cannot be used in games."
```

Comment:
This could also just return an empty list, whatever is perferable

Reply:
Please return an empty list here.